stack.yaml.ghcjs

```diff
@@ -15,5 +15,7 @@ extra-deps:
   commit: e65815417ffd464bfb6d5444f7ff1ad4778a9417
 - github: julian-berbel/language-python
   commit: cbdf9b33cc3c30e66c8455165ed1c7d70a300a3e
+- github: mrkkrp/text-metrics
+  commit: 4b2b8c4613d4e948851bc81d823f9e158aadd6c3
```

Comment:
It looks like the original implementation of this package was C-based and it is the probable cause of not being part of the ghcjs distro. Nevertheless, I had to manually add it it order JS runtime to work.